civihr_default_theme/scss/partials/_navs.scss

```diff
@@ -1,28 +1,34 @@
 // Styles for Navs
 // --------------------------------------------------
 
+.nav > li > a {
+  background-color: $light;
+}
+
 .nav-tabs-header {
-    &.nav-justified {
-        box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.035), 0 4px 8px 0 rgba(0, 0, 0, 0.035);
-        margin-bottom: 20px;
+  &.nav-justified {
+    box-shadow: 0 3px 10px 0 rgba(0, 0, 0, 0.035), 0 4px 8px 0 rgba(0, 0, 0, 0.035);
+    margin-bottom: 20px;
 
-        li {
-            a {
-                font-size: 18px;
-                color: $gray-darker;
-                border: 0;
-                border-bottom: 3px solid transparent;
-            }
+    li {
+      a {
+        font-size: 18px;
+        color: $gray-darker;
+        border: 0;
+        border-bottom: 3px solid transparent;
```

Comment:
@AkA84 I mean use a variable, like,
`$crm-box-shadow-color: rgba(0, 0, 0, 0.035)` , 
`box-shadow: 0 3px 10px 0 $crm-box-shadow-colo, 0 4px 8px 0 $crm-box-shadow-color`

Similar for `border-bottom` color